easypost/requestor.py

```diff
@@ -51,18 +52,41 @@ def _objects_to_ids(cls, param: Dict[str, Any]) -> Dict[str, Any]:
         else:
             return param
 
+    @classmethod
+    def form_encode_params(cls, data: dict, parent_keys: List[str] = None, parent_dict: dict = None) -> dict:
```

Comment:
These **MUST** be capital `Dict` because we support Python 3.6/7/8 which do not allow the native `dict` type as a type hint. You'll need to use `typing.Dict` until we support Python 3.9+. This will fail at runtime otherwise.

```suggestion
    def form_encode_params(cls, data: Dict, parent_keys: List[str] = None, parent_dict: Dict = None) -> Dict:
```

Reply:
resolved